src/protobufs/src/stripe-service/stripe-service.proto

```
@@ -12,13 +12,57 @@ service StripeService {
   rpc CreateAndPostTransfer(CreateAndPostTransferRequest) returns (Empty);
   rpc VoidPendingTransfer(VoidPendingTransferRequest) returns (Empty);
   rpc CreatePendingTransfer(CreatePendingTransferRequest) returns (Empty);
+  rpc GetStripeAccountId(GetStripeAccountIdRequest) returns (GetStripeAccountIdResponse);
+  rpc SetStripeAccountId(SetStripeAccountIdRequest) returns (Empty);
```

Buf (1.62.1)

"service.stripe.Empty" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)